codecov.yml

```diff
@@ -1,3 +1,8 @@
+coverage:
+    range: 90..100
```

Comment:
The current lower bound on coverage is 95%: https://github.com/OpenMined/PySyft/blob/master/.github/workflows/tests.yml#L69

Reply:
Done